app/presenters/two_factor_authentication/phone_selection_presenter.rb

```diff
@@ -2,7 +2,7 @@ module TwoFactorAuthentication
   class PhoneSelectionPresenter < SelectionPresenter
     def type
       if MfaContext.new(configuration&.user).phone_configurations.many?
-        "#{super}_#{configuration.id}"
+        "#{super}:#{configuration.id}"
```

Comment:
This needs to be `_` rather than `:` due to an issue Moncef found. The controller that splits on `:` needs to separate on the last `_`.

Reply:
Yes.  I tried that and it didn't work because we have keys like this: my_key.  However I realized the pickle we are in last nite so I'm simply going to rename those keys.

Reply:
Or a regex - whichever is least confusing for everyone else.